### PR TITLE
React Calculator Interactions

### PR DESCRIPTION
**Pull request by kwizl**
# Project Interactions

- In this milestone, I will be tying the calculations module with the components and adding interactions to the buttons by using event handlers.

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -10,15 +11,21 @@ const ButtonPanel = () => {
     'group-5': ['0', '.', '='],
   };
 
+  const handleClick = buttonName => {
+    onClick.clickHandler(buttonName);
+  };
```
Comment (wathigo) [inline src/components/ButtonPanel.js:16]: **REQUIRED**
* `onClick` is function not an object. We can re-write this as:
```js
  const handleClick = buttonName => {
    onClick.clickHandler(buttonName);
  };
```

**Review by wathigo — CHANGES_REQUESTED** [review]
## Changes Required :o:  
Great job on this milestone.
Let's fix the issue pointed out in the comment :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

File: src/components/ButtonPanel.js (hunk 1 above)
Comment (kwizl) [reply src/components/ButtonPanel.js:16]: Morning Simon. I haven't understood the correction that you have suggested.

**Review by whiz25 — APPROVED** [review]
## Approved :clap:
Nice work @kwizl . You made the requested change with the `ButtonPanel.js` file as requested.
Merge and proceed to the next project.


### Martin Afani

**Pull request merged**